ui/appui-react/src/appui-react/popup/PopupManager.tsx

```diff
@@ -49,6 +51,17 @@ export interface PopupInfo {
   parentDocument: Document;
 }
 
+/**
+ * @public
```

Comment:
nit:
I'm not sure if we could have a rule for linter, but please keep this in one line: `/** @public */`
In other places prefer to add content in first line.
```
/** This component...
 * @internal
 */
```

Reply:
Will do thanks. I'm sure we could add a rule if we think it is worth it.

Reply:
Fixed.